packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.tsx

```diff
@@ -49,11 +50,17 @@ export class MessageBar extends React.Component<IMessageBarProps, IMessageBarSta
     /**
      * Live regions need an update to announce content.
      */
-    setTimeout(() => {
+    this._mountTimeout = setTimeout(() => {
```

Comment:
Instead of using `setTimeout`, but instead you can use `this._async.setTimeout`. Then you don't have to worry about clearing the timeout; it will do it for you!